### DISCUSSION TITLE
File cards in NoteCards

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1835] [General]
From the paper [Hypertext Habitats: Experiences of Writers in NoteCards](https://dl.acm.org/doi/pdf/10.1145/317426.317435):

> The boundaries of a notefile can be relaxed further using *file* cards. A file card is a card whose substance is simply the name of a file. When a link is followed to a file card, NoteCards finds the file with that name and opens it in a text editor window. File cards were designed to allow work to exist in a file external to the notefile, and yet still be made available from within the notefile.

Does NoteCards still support file cards? How are file cards created? The documentation has no reference and the card creation menu doesn't list the file type.

**Comment by nbriggs** [comment]
Have you loaded library/NCFILECARD.LCOM?  NoteCards is an extensible system, you can load extensions that dynamically add to the card types...

**Comment by pamoroso** [comment]
Loading library/NCFILECARD.LCOM did the trick, thanks. I assumed the file card was in the base system.
